packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,11 +6,25 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## December
 
+### Weekly release 2022.12.2
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.12.2](#bugs-fixed-in-2022-12-2)                      |December 21, 2022            |
+
+#### Bugs fixed in 2022.12.2
+
+* UPDATE requests to the `/brands/{brandId}/pages/sign-in/preview` and `/brands/{brandId}/pages/sign-in` endpoints sometimes returned a `null` response that resulted in an HTTP 400 Bad Request error. (OKTA-551632)
```

Comment:
```suggestion
* PUT requests to the `/brands/{brandId}/pages/sign-in/preview` and `/brands/{brandId}/pages/sign-in` endpoints under certain conditions returned a `null` response body with an HTTP 400 Bad Request error. (OKTA-551632)
```
Sorry, I know this has already been reviewed, but you shouldn't capitalize UPDATE unless it's a REST request operations? Maybe use "Update requests to the ..." or use "PUT requests to the ..."?
Also, the null response doesn't "resulted" an 400 bad request error? Please confirm with dev.